### ISSUE TITLE
feat: make PostCSS optional…possibly off by default

### ISSUE THREAD

**Issue by jaredcwhite** [bridgetownrb/bridgetown#862]
Two things have happened in the last year or two:

* Vanilla CSS has gotten [really, really good](https://www.spicyweb.dev/css-nouveau/) (shameless plug!).
* esbuild has gotten better at transpiling certain CSS features for older browsers should you need that (like you can use nesting and have it flattened out). See: https://esbuild.github.io/content-types/#css

Because of this, and because I rarely reach for any particular PostCSS plugins in my own builds any more—in fact more and more my CSS is contained within declarative shadow DOM which I don't put through any processor—it's starting to feel like PostCSS is unnecessary complexity. Like Sass, it's a product of a bygone age.

So I find myself mulling over the idea of making PostCSS optional. And if that's a given, then the question becomes: do we truly make it an option? A "bundled configuration" perhaps? And not the default?

I'd love feedback from the community on this! If we go down this road, it makes the most sense to do it for Bridgetown 2.0. Otherwise I'd want to wait and revisit for Bridgetown 3.0.

**Comment by ayushn21** [comment]
> I find myself mulling over the idea of making PostCSS optional.

I'm always in favour of something that removes a build step.

> do we truly make it an option? 

I think we should still have it baked in so it could be set up with a flag passed into `bridgetown new`, but a default setup wouldn't have it included. I reckon that's the path we should take ....

**Comment by brandondrew** [comment]
I think as long as it is _easy by default_ then _off by default_ is not going to cause anyone any pain.

**Comment by jaredcwhite** [comment]
@brandondrew @ayushn21 right, I'm definitely in favor of keeping it an easy option, but maybe we don't include it by default on every `bridgetown new`.

**Comment by nogweii** [comment]
Chiming in that I'd love to disable PostCSS in my new website, as I don't use it's features at all.
